test/lit/passes/translate-to-new-eh.wast

```diff
@@ -94,6 +134,21 @@
   ;; CHECK-NEXT:   )
   ;; CHECK-NEXT:  )
   ;; CHECK-NEXT: )
+  ;; STACKIR-OPT:      (func $try-none-tag-none-with-rethrow (type $1)
+  ;; STACKIR-OPT-NEXT:  (local $0 exnref)
+  ;; STACKIR-OPT-NEXT:  block $outer0
+  ;; STACKIR-OPT-NEXT:   block $catch_all2 (result exnref)
+  ;; STACKIR-OPT-NEXT:    block $catch1 (result exnref)
+  ;; STACKIR-OPT-NEXT:     try_table (catch_ref $e-empty $catch1) (catch_all_ref $catch_all2)
+  ;; STACKIR-OPT-NEXT:      call $foo
+  ;; STACKIR-OPT-NEXT:     end
+  ;; STACKIR-OPT-NEXT:     br $outer0
+  ;; STACKIR-OPT-NEXT:    end
+  ;; STACKIR-OPT-NEXT:    throw_ref
+  ;; STACKIR-OPT-NEXT:   end
+  ;; STACKIR-OPT-NEXT:   throw_ref
+  ;; STACKIR-OPT-NEXT:  end
+  ;; STACKIR-OPT-NEXT: )
```

Comment:
Nice, looks like this is an example of those `local.set/get` operations getting removed.